manifests/config/project.pp

```diff
@@ -55,7 +55,8 @@
   $node_executor_settings = {},
   $projects_dir           = undef,
   $resource_sources       = $rundeck::resource_sources,
-  $scm_import_properties  = $rundeck::scm_import_properties,
+  $scm_import_properties  = {},
```

Comment:
@jyaworski - yes, they break in the current implementation.  In the template, the hash is iterated over like this:
```
<%- @scm_export_properties.sort.each do |k,v| -%>
<%= k %> = <%= v %>
<%- end -%>
```
If the hash is empty (which it may be if the user does not have a need to configure the SCM for a certain project), provisioning would fail on a null hash if it was not defaulted to '{}' here.